test/libsolidity/semanticTests/array/push/push_no_args_bytes.sol

```diff
@@ -21,9 +21,9 @@ contract C {
 // ----
 // l() -> 0
 // g(uint256): 70 ->
-// gas irOptimized: 183587
-// gas legacy: 183811
-// gas legacyOptimized: 179218
+// gas irOptimized: 187549
```

Comment:
This is the only case in our tests that increases in irOptimized costs - and looking at the diff, I think it's actually an inlining issue, respectively we're missing an optimizer step that removes function return variables, if they are simply assigned a constant value in the function...